src/utils/CoinbasePixel.ts

```diff
@@ -231,47 +236,35 @@ export class CoinbasePixel {
 
   /** Failed to load the pixel iframe */
   private onFailedToLoad = () => {
-    const message = 'Failed to load CB Pay pixel. Falling back to opening in new tab.';
     this.state = 'failed';
-    console.warn(message);
-    this.onReadyCallback?.(new Error(message));
-    this.runQueuedOpenExperience();
-  };
 
-  /** Run any queued open experience options. Note: the window.open may not work outside of a click event and fail on browsers like Safari. */
-  private runQueuedOpenExperience = () => {
-    if (this.queuedOpenOptions) {
-      this.log('Running queued experience');
-      const options = { ...this.queuedOpenOptions };
-      this.queuedOpenOptions = undefined;
-      this.openExperience(options);
+    // If a fallback option is provided we only want to provide a warning since we can still attempt to open the widget
+    if (this.onFallbackOpen) {
+      if (this.debug) {
+        console.warn('Failed to load CB Pay pixel. Falling back to opening in new tab.');
```

Comment:
nit: Still good to warn if `!this.onFallbackOpen`.
```suggestion
    if (this.debug) {
      console.warn('Failed to load CB Pay pixel.' + this.onFallbackOpen ? ' Falling back to opening in new tab.' : '');
    }

    if (this.onFallbackOpen) {
```

Reply:
Added an debug error log.